.travis.yml

```diff
@@ -46,7 +46,7 @@ before_install:
 
 # Install dependencies
 install:
-  - opam pin add --no-action ./sources
+  - opam pin add --no-action ./src
```

Comment:
Isn't the goal of this PR that the root of the repo can be pinned instead of the source directory ?